### ISSUE TITLE
After updating to v1.49.1 the HPB service shows down

### ISSUE THREAD

**Issue by suxlex** [nextcloud/nextcloudpi#1565]
After updating from NextCloudPi v1.49.0 to v1.49.1, the "HPB service = down" is displayed in red under System Info.
If I go back to version 1.49.0, then again "HPB service = up" is in green.

**System information**

<details>
<summary>NextCloudPi diagnostics</summary>

```

NextCloudPi version  v1.49.0
NextCloudPi image    NextCloudPi_07-28-22
OS                   Debian GNU/Linux 11. 5.10.0-18-amd64 (x86_64)
automount            no
USB devices          none
datadir              /var/www/nextcloud/data
data in SD           yes
data filesystem      ext2/ext3
data disk usage      13G/29G
rootfs usage         13G/29G
swapfile             /var/swap
dbdir                /var/lib/mysql
Nextcloud check      ok
Nextcloud version    24.0.3.2
HTTPD service        up
PHP service          up
MariaDB service      up
Redis service        up
HPB service          up
Postfix service      up
internet check       ok
port check 80        open
port check 443       open
IP                   ***REMOVED SENSITIVE VALUE***
gateway              ***REMOVED SENSITIVE VALUE***
interface            ens18
certificates         ***REMOVED SENSITIVE VALUE***
NAT loopback         no
uptime               1:27

```
</details>

<details>
<summary>Nextcloud configuration</summary>

```

{
    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": {
            "0": "localhost",
            "1": "192.168.1.80",
            "2": "cl.halpixel.de",
            "11": "2003:c1:6f01:d600:dc94:63ff:feb1:6e8c",
            "3": "cl.halpixel.de",
            "14": "nextcloudpi"
        },
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "24.0.3.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "default_phone_region": "DE",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 0,
            "timeout": 0,
            "password": "***REMOVED SENSITIVE VALUE***"
        },
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "loglevel": "2",
        "log_type": "file",
        "maintenance": false,
        "theme": "",
        "simpleSignUpLink.shown": false,
        "htaccess.RewriteBase": "\/",
        "updater.release.channel": "stable",
        "data-fingerprint": "9b4229aa47789d3a2d81b81ea9b99a82",
        "mail_sendmailmode": "smtp",
        "jpeg_quality": "60",
        "app_install_overwrite": [
            "calendar",
            "admin_notifications",
            "documents"
        ],
        "has_rebuilt_cache": true,
        "mail_smtpsecure": "tls",
        "mail_smtpport": "587",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "https:\/\/cl.halpixel.de\/",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "tempdirectory": "\/var\/www\/nextcloud\/data\/tmp",
        "logfile": "\/var\/www\/nextcloud\/data\/nextcloud.log"
    }
}

```
</details>

**Comment by mbousq** [comment]
Did you reboot after upgrading ? I upgraded ncp to v1.50 and my Nextcloud instance to 24.0.5 and I had "HPB service = down". A simple reboot fixed this :p

**Comment by suxlex** [comment]
Reboot - yes, I think so. 
Yesterday I installed v1.50 and HPB is still up - my problem is now gone.
Thanks for pointing out the restart that may be needed!

**Comment by sunjam** [comment]
Same with docker.  Admin must stop and restart their container to bring HPB up.

**Comment by theCalcaholic** [comment]
Fix will be rolled out with v1.50.1

**Comment by sunjam** [comment]
Update, on Docker HPB stays down after pulling latest `v1.50.1` and will no longer reboot upon `docker compose stop nextcloudpi && docker compose start nextcloudpi`

**Comment by ZendaiOwl** [comment]
I think this is resolved, see #1590 
Activate the instance before doing any restart and allow the init to fully complete, see if that works?

**Comment by theCalcaholic** [comment]
Yeah, this *should* be resolved (note though, that HPB is expected to fail if you have not used a url with e.g. letsencrypt and did not supply one as parameter to the container)
